rlclientlib/logger/http_transport_client.cc

```diff
@@ -58,7 +55,7 @@ namespace reinforcement_learning {
       }
 
       // If the response is not the expected code then it has failed. Retry if possible otherwise report background error.
-      if(code != status_codes::Created) {
+      if(code != status_codes::Created && code != status_codes::NoContent) {
```

Comment:
is NoContent considered as success?

Reply:
Yes
HTTP Status 204 (No Content) indicates that the server has successfully fulfilled the request and that there is no content to send in the response payload body.